salt/cloud/clouds/profitbricks.py

```diff
@@ -120,7 +120,7 @@
     import profitbricks
     from profitbricks.client import (
         ProfitBricksService, Server,
-        NIC, Volume, FirewallRule,
+        NIC, Volume, FirewallRule, IPBlock,
```

Comment:
Is `IPBlock` a new addition to this upstream library or has it been present for a while? We want to make sure we preserve backward compatibility.